kbase-extension/scss/partials/_errorDialog.scss

```diff
@@ -0,0 +1,30 @@
+/* Tabbed error dialog */
+.kb-error-dialog {
```

Comment:
Move inline styles to the SCSS files. Code display is consistent with what is in the error logs:

<img width="697" alt="Screenshot 2021-01-25 at 09 31 31" src="https://user-images.githubusercontent.com/556057/105742273-23294000-5ef0-11eb-96fa-50a4881fd7cc.png">